sql/parser_test.go

```diff
@@ -39,6 +39,19 @@ COLUMN
   cross(embedding(emplyoee.name), bucketize(last_name, 1000))
 LABEL employee.salary
 INTO sqlflow_models.my_dnn_model;
+`
+	testMultiColumnTrainSelect = testStandardSelectStmt + `TRAIN DNNClassifier
+WITH
+  n_classes = 3,
+  hidden_units = [10, 20]
+COLUMN
+  employee.name,
+  bucketize(last_name, 1000),
+  cross(embedding(emplyoee.name), bucketize(last_name, 1000))
```

Comment:
This COLUMN clause doesn't have the FOR keyword. Is that alright?

Reply:
That's alright, the keyword of `FOR` is optional.